cypress/support/fragments/marcAuthority/marcAuthority.js

```diff
@@ -87,6 +88,13 @@ export default {
     cy.do(Button('Edit').click());
     QuickMarcEditorWindow.waitLoading();
   },
+  delete: () => {
+    cy.do([
+      rootSection.find(Button('Actions')).click(),
+      Button('Delete').click(),
+      Modal('Confirm deletion of authority record').find(Button('Delete')).click(),
+    ]);
+  },
```

Comment:
There are already existing methods to delete marc authority record in marcAuthoritiesDelete.js file please use them.